test/common/network/socket_option_test.h

```diff
@@ -65,8 +65,8 @@ class SocketOptionTest : public testing::Test {
   TestThreadsafeSingletonInjector<Api::OsSysCallsImpl> os_calls_{[this]() {
     // Before injecting OsSysCallsImpl, make sure validateIpv{4,6}Supported is called so the static
     // bool is initialized without requiring to mock ::socket and ::close.
-    std::make_unique<Address::Ipv4Instance>("1.2.3.4", 5678);
-    std::make_unique<Address::Ipv6Instance>("::1:2:3:4", 5678);
+    (void)std::make_unique<Address::Ipv4Instance>("1.2.3.4", 5678);
+    (void)std::make_unique<Address::Ipv6Instance>("::1:2:3:4", 5678);
```

Comment:
Not a huge fan of this one; I think we had some thread around trying to get to the bottom of inconsistency here; who has the potato?